examples/src/main/java/io/github/tcdl/examples/MultipleRequesterResponder.java

```diff
@@ -12,9 +12,12 @@
 /**
  * Created by anstr on 6/9/2015.
  */
-public class MultipleRequesterResponder extends BaseExample {
+public class MultipleRequesterResponder {
+
     private static final Integer NUMBER_OF_RESPONSES = 1;
 
+    private MsbTestHelper util = MsbTestHelper.getInstance();
```

Comment:
Maybe "private MsbTestHelper msbTestHelper = MsbTestHelper.getInstance();" will be better?